auth/azure.go

```diff
@@ -148,3 +168,25 @@ func ReadAzureProfile(path string) (*AzureProfile, error) {
 
 	return &ap, nil
 }
+
+func DecodeUtfbomFile(contents []byte) ([]byte, error) {
```

Comment:
Why not use https://godoc.org/golang.org/x/text/encoding/unicode

Reply:
We already have `dimchansky/utfbom` as a vendored dependency because of Azure, switching to `x/text/encoding/unicode` will also require `x/text/transform` neither of which are currently vendored.

Do you have a preference between the two?